### ISSUE TITLE
Discover projection classes [DATAGRAPH-1183]

### ISSUE THREAD

**Issue by spring-issuemaster** [spring-projects/spring-data-neo4j#1745]
**[Gerrit Meier](https://jira.spring.io/secure/ViewProfile.jspa?name=gerrit.meier)** opened **[DATAGRAPH-1183](https://jira.spring.io/browse/DATAGRAPH-1183?redirect=false)** and commented

It should be possible to use a projection class without manually defining it as a `@QueryResult`

---

1 votes, 2 watchers

**Comment by meistermeier** [comment]
This is not needed anymore.

**Comment by adetrie** [comment]
@meistermeier may you explain how to achieve it ? (I work with ReactiveNeo4jRepository and spring-webflux) 
Thx

**Comment by meistermeier** [comment]
@adetrie It's in the docs https://docs.spring.io/spring-data/neo4j/docs/current/reference/html/#projections
But please do not hesitate to reach out if there are still questions.

**Comment by ThomHurks** [comment]
@meistermeier Maybe the docs could be improved a bit regarding projections. I copied the interface projections example code from the docs located at https://docs.spring.io/spring-data/neo4j/docs/current/reference/html/#projections.sdn.full-example and got the error `No property 'findAllInterfaceProjections' found for type 'TestEntity'`. I then found your (I think) comment at https://community.neo4j.com/t5/drivers-stacks/reactiveneo4jrepository-no-property-found-for-type/m-p/36455/highlight/true#M2765 which indicates the method name should be `findAllProjectedBy` and this indeed works.
